docs/tutorial/installation.rst

```diff
@@ -38,6 +40,9 @@ http://localhost:15672/. The username is ``guest`` and the password is
 read the messages in the queues. Keep it open in a browser tab, we'll need it
 later.
 
+If your project uses containers, consult the `RabbitMQ documentation`_ on it's conatiner.
+
+.. _RabbitMQ documentation: https://www.rabbitmq.com/download.html#docker?
```

Comment:
There is no question mark at the end of the URL if you want to get to the proper target on the page.